ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -327,4 +347,42 @@ private static boolean bug1733118Affected(String namespace, String docType, Obje
             .matches("0[.]([0-9]|1[0-6])[.].*"); // < 0.17
   }
 
+  // See bug 1751753 for explanation of context.
+  private static void processForBug1751753(ObjectNode json) {
+    // Sanitize keys in the SEARCH_COUNTS histogram.
+    JsonNode searchCounts = json.path("payload").path("keyedHistograms").path("SEARCH_COUNTS");
```

Comment:
Can we be guaranteed of the capitalization here? The BQ ingestion logic lowercases all field names, so we tolerate casing differences from that perspective. But the logic as written here would miss sanitizing the field if the histogram were recorded as "search_counts" in the JSON.

Maybe this is something @chutten can comment on.

Reply:
I confirm that histogram names are SCREAMING_SNAKE_CASE when they leave Firefox Desktop. Scalars and Events are different, as are all other parts of other ping payloads.

Reply:
@chutten indicates that histogram names are reliably SCREAMING_SNAKE_CASE in what the browser sends.